pandas/core/frame.py

```diff
@@ -420,6 +421,7 @@ def _constructor(self) -> Type[DataFrame]:
     _constructor_sliced: Type[Series] = Series
     _deprecations: FrozenSet[str] = NDFrame._deprecations | frozenset([])
     _accessors: Set[str] = {"sparse"}
+    __hash__ = NDFrame.__hash__
```

Comment:
was this defined before?

Reply:
ATM this PR puts OpsMixin ahead of NDFrame in the MRO, which is why this needs to be re-pinned.  flipping those should make this unnecessary